frontend/badges/cppCourseGraduate/query.sql

```diff
@@ -0,0 +1,48 @@
+SELECT
+	DISTINCT p.user_id
+FROM
+    (
+        SELECT
+            a.alias,
+            a.assignment_id,
+            psp.problem_id,
+            i.user_id,
+            sum((r.contest_score)) as best_score_of_problem
```

Comment:
Esto debería ser `MAX(...)`, no?

```suggestion
            MAX(r.contest_score) AS best_score_of_problem
```

si no, alguien que mande 1000 envíos al mismo problema se puede considerar que resolvió más del 60%!